packs/fireeye/actions/submit_malware.py

```diff
@@ -0,0 +1,16 @@
+from lib.actions import BaseAction
+import json
+import os
+
+
+class SubmitMalwareAction(BaseAction):
+    def run(self, **kwargs):
```

Comment:
Yes sir. Those do need to be in the payload. They are set as mandatory in the action metadata.